open_academy/models/partner.py

```diff
@@ -0,0 +1,8 @@
+from odoo import models, fields
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    instructor = fields.Boolean(default=False)
```

Comment:
Boolean fields are `False` by default, the `default` attribute it's not necessary.